docs/src/openapi-v1.yaml

```diff
@@ -367,6 +367,68 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AccountValidation'
+  /accounts/{accountId}/convert:
+    get:
+      tags:
+      - accounts
+      summary: Convert a given account Id to a SS58 address.
```

Comment:
```suggestion
      summary: Convert a given AccountId to a SS58 address.
```

Reply:
The idea being we should set all the AccountId's to be the same in this portion of the docs. We also will need to rebuild the docs after updating these changes.